tests/components/DrugOrderEntry/addForm/AddForm.test.jsx

```diff
@@ -343,13 +343,13 @@ describe('handleSubmitDrugForm() method', () => {
 
   it('should change the selected order after a drug has been added', () => {
     const component = getComponent();
-    props.setSelectedOrder.mockReset();
+    props.setSelectedOrderAction.mockReset();
     component.setProps({
       createOrderReducer: {
         addedOrder: { order: 'just-a-sample-order' },
         status: { added: true, errror: false },
       },
     });
-    expect(props.setSelectedOrder).toHaveBeenCalledTimes(1);
+    expect(props.setSelectedOrderAction).toHaveBeenCalledTimes(1);
```

Comment:
you can destructure here too.  const { setOrder, selectDrugSuccessAction } = props

Reply:
done